packages/Search/src/details/DTK_DetailsTreeTraversal.hpp

```diff
@@ -66,7 +66,10 @@ struct TreeTraversal
     KOKKOS_INLINE_FUNCTION
     static Node const *getRoot( BVH<DeviceType> bvh )
     {
-        return bvh._internal_nodes.data();
+        if ( bvh._internal_nodes.extent( 0 ) > 0 )
+            return bvh._internal_nodes.data();
+        else
+            return bvh._leaf_nodes.data();
```

Comment:
Shouldn't you further separate? Unless Kokkos views initialize internal pointers to `nullptr`?
```c++
if ( bvh._internal_nodes.extent( 0 ) > 0 )
  ...
else if ( bvh._leaf_nodes.extent( 0 ) > 0 )
  ...
else
  return nullptr;
```